src/App.js

```diff
@@ -1,19 +1,22 @@
-import React from 'react';
-import Calculator from './components/calculator';
+/* eslint-disable react/jsx-key */
+/* eslint-disable import/extensions */
```

Comment:
Please do not disable linters in every component and fix errors raised by linters instead.

Reply:
Hey Sajjad! I have tried working on this error but I can't see it locally and it's only appearing on GitHub kindly assist me to fix it. Additionally, I have tried looking for answers online and from folks at microverse and we have all failed.